en/engine/renderable/model-component.md

```diff
@@ -0,0 +1,62 @@
+
+### Instancing batching
+
+The batch through **instancing** is suitable for drawing a large number of dynamic models with the same vertex data. When enabled, drawing will be grouped according to the material and vertex data, and the instanced attributes information will be organized in each group, and then complete the drawing at one time.
+
+  > **Note**: For the support and related settings of the skinning model, refer to the [Skeletal Animation Component](../animation/skeletal-animation.md#AboutDynamic-Instancing) documentation.
+
+
+In addition, inside each group, the instanced attributes supports custom additional instanced attributes, which can pass more per-instance data between different instances (such as the difference in appearance of a diffuse color between different characters, or the influence of wind in a large grass field). This requires the support of custom effects. For more detailed instructions, please refer to the [Syntax Guide](../../material-system/effect-syntax.md#Custom-Instanced-Properties) documentation.
+
+### VB-merging batching
+
+__VB-merging batching__ is suitable for drawing a large number of non-skinned dynamic models with low number of faces and different vertex data. When enabled, drawing will be grouped according to the material, and then the vertex and world transformation information will be merged in each frame of each group, and then completed in batches. <sup id="a1">[1](#f1)</sup>.
+
+Operations such as merging vertices per frame introduce a portion of CPU overhead, which is particularly expensive in JavaScript. In addition, it is necessary to remind that the number of __draw calls__ is not as low as possible. Although, it is important to note, minimizing the number of draw calls to the extreme doesn't necessarily lead to the best (or even good) performance <sup id="a2">[2](#f2)</sup>. Optimal performance is often the result of CPU and GPU load balancing, so when using batch functions, be sure to do more tests to identify performance bottlenecks and do targeted optimization.
+
+## Batch best practices
+
+Generally speaking, the priority of the batch system is: *static batching* -> *dynamic instancing* -> *dynamic batching*. 
+
+The material must be insured that it is consistent, under this premise:
+ - If you are certain that certain models will remain completely static during the game cycle, use *static batching*. 
+ - If there are a large number of the same model repeated drawing, there is only a relatively controllable small difference between each other, use *dynamic instancing*. 
+ - If there are a large number of models with very low number of triangles but different vertex data, consider trying *dynamic batching*.
```

Comment:
Love how the tone manner is conveyed accurately as the original doc here. Very important.